extensions/ql-vscode/src/vscode-tests/no-workspace/history-item-label-provider.test.ts

```diff
@@ -95,8 +95,8 @@ describe('HistoryItemLabelProvider', () => {
       expect(labelProvider.getLabel(fqi)).to.eq(`${dateStr} query-name (javascript) github/vscode-codeql-integration-tests in progress %::${dateStr} query-name (javascript) github/vscode-codeql-integration-tests in progress %`);
     });
 
-    it('should interpolate query when not user specified', () => {
-      const fqi = createMockRemoteQueryInfo();
+    it('should interpolate query when user not specified', () => {
```

Comment:
Maybe I'm reading this incorrectly, but this change seems wrong 🤔 We're testing the case where the label is not user specified (not whether the user is specified).

The original is correct, though could probably be clearer! 🙈

Reply:
Thanks @shati-patel! I have gone with `should interpolate query when not user-specified` in the end.